option optimize_for = LITE_RUNTIME;

message TreasureDressRequest
{
 repeated uint32 pos = 1;
 repeated uint32 posZhengXing = 2;
}
